atomic_reactor/plugins/build_source_container.py

```diff
@@ -29,8 +30,11 @@ def run(self):
         Returns:
             BuildResult
         """
-        source_data_dir = tempfile.mkdtemp()  # TODO: from pre_* plugin
-        # TODO fail when source dir is empty
+        fetch_sources_result = self.workflow.prebuild_results.get(PLUGIN_FETCH_SOURCES_KEY, {})
+        source_data_dir = fetch_sources_result.get('image_sources_dir')
+        if not os.path.isdir(source_data_dir) or not os.listdir(source_data_dir):
+            # TODO fail when source dir is empty
```

Comment:
We can remove this TODO as we agreed that empty source container image is still valid